### DISCUSSION TITLE
a incomplete Liverworts cp genome

### DISCUSSION THREAD

**Discussion by zgszm** [Kinggerm/GetOrganelle#153] [Q&A] [answered]
Hi, I am assembling a liverworts chloroplast genome with the recommended commands but the result is incomplete. I have optimized the parameter according to ### What should I do with incomplete result/"broken assembly graph"? 
The result are still incomplete. I hope you can help to accomplish it.
The get_org.log.txt and the assembly graph are following :
![yy27](https://user-images.githubusercontent.com/96463605/159820295-0dafdc8d-8587-4001-b179-28d4dc39a6cb.png)
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8337536/get_org.log.txt)

**Comment by Kinggerm** [comment, marked as answer]
Given that you have tried parameters tuning, the hope for completeness is low.
You may compare the current assembly to a closely related sample and see what you missed. I believe it's nearly complete.

**Comment by zgszm** [comment]
The sample had been assmbled successfully. However, now it is not assembled successfully.

**Reply by Kinggerm** [reply]
For shallow coverage samples, the result may be vulnerable to some parameters. Since you had it successfully before, could you check the historical parameters and versions?

**Comment by zgszm** [comment]
I use same parameters. The version of getorganelle was 1.7.5.0

**Reply by Kinggerm** [reply]
Please always provide the log file

**Reply by zgszm** [reply]
I haven't found the original log file.

**Comment by zgszm** [comment]
Hi, i assmble successfully it with getorganelle1.7.5.0. I adjest the parameter "-w" up to "-w 90". It turned out to be "-w 86" estimated by program.

**Reply by Kinggerm** [reply]
Great!
Yeah, I know. Tuning parameters can be tricky.
Thanks for the update!
